src/data/monsterparts.txt

```diff
@@ -2274,3 +2274,6 @@
 2451	herd of archaea	another archaea	archaea	different archaea
 2452	yeast gang	nodule
 2453	Embering Hulk	arm	ember	flames	head	leg
+2475	feral bunny	scales
```

Comment:
how did you get these? They don't match the wiki.

Reply:
When I ran missingManuel, which visited all Manuel pages, it mentioned a mismatch.
Which is strange, since I don't see it in MonsterManuelManager.java.
I removed the lines from monsterparts.txt and it no longer shows it.
Weird.
I'll remove that change.